.circleci/config.yml

```diff
@@ -840,7 +840,7 @@ jobs:
     <<: *machine-linux-2xlarge
     environment:
       <<: *env-linux-2xlarge
-      GCLIENT_EXTRA_ARGS: '--custom-var=checkout_mac=True --custom-var=host_os=mac'
+      GCLIENT_EXTRA_ARGS: '--custom-var=checkout_mac=True --custom-var=host_os=mac --custom-var=check_gn_version=False'
```

Comment:
The variable and corresponding patch `disable_ensure_gn_version_gclient_hook.patch` can be removed now that we have `patch_the_ensure_gn_version_py_script_to_work_on_mac_ci.patch` that directly patches the script.

Reply:
```suggestion
      GCLIENT_EXTRA_ARGS: '--custom-var=checkout_mac=True --custom-var=host_os=mac'
```